LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2014 Stanislav Vishnevskiy
+Copyright (c) 2014-present, Stanislav Vishnevskiy
```

Comment:
I'd have to double check on this, could you remove the change for now?

Reply:
> I'd have to double check on this, could you remove the change for now?

Done.

Reply:
@mitchellhenke It seems that the use of copyright year `2014-present` is not recommended, see https://github.com/riverrun/comeonin/pull/153#issuecomment-757405132.

Do let me know if you found something on this as well. Thanks. :smile:

Reply:
Thanks for the reference, everything looks great now 🙂